tests/test_common_cells.hpp

```diff
@@ -149,6 +172,76 @@ inline cell make_cell_ball_and_3stick(bool with_stim = true) {
 }
 
 /*
+ * Create 'soma-less' cell with single cable, with physical
+ * parameters from Rallpack 1 model.
+ *
+ * Common properties:
+ *    mechanisms: passive
+ *        membrane conductance: 0.000025 S/cm² ( =  1/(4Ω·m²) )
+ *        membrane reversal potential: -65 mV (default)
+ *    diameter: 1 µm
+ *    length: 1000 µm
+ *    bulk resistivity: 100 Ω·cm
+ *    capacitance: 0.01 F/m² [default]
+ *    compartments: 4
+ *
+ * Stimulus:
+ *    end of dendrite, t=[0 ms, inf), 0.1 nA
+ *
+ * Note: zero-volume soma added with same mechanisms, as
+ * work-around for some existing fvm modelling issues.
+ *
+ * TODO: Set the correct values when parameters are generally
+ * settable! 
+ *
+ * We can't currently change leak parameters
+ * from defaults, so we scale other electrical parameters
+ * proportionally.
+ */
+
+inline cell make_cell_simple_cable(bool with_stim = true) {
+    cell c;
+
+    c.add_soma(0);
+    c.add_cable(0, segmentKind::dendrite, 0.5, 0.5, 1000);
+
+    double r_L  = 100;
+    double c_m  = 0.01;
+    double gbar = 0.000025;
+    double I = 0.1;
+
+    // fudge factor! can't change passive membrane
+    // conductance from gbar0 = 0.001
+
+    double gbar0 = 0.001;
+    double f = gbar/gbar0;
+
+    // scale everything else
+    r_L *= f;
+    c_m /= f;
+    I /= f;
+
+    for (auto& seg: c.segments()) {
+        seg->add_mechanism(pas_parameters());
+        seg->mechanism("membrane").set("r_L", r_L);
+        seg->mechanism("membrane").set("c_m", c_m);
+        // seg->mechanism("pas").set("g", gbar);
+ 
+        if (seg->is_dendrite()) {
+            seg->set_compartments(4);
+        }
+    }
+
+    if (with_stim) {
+        // stimulus in the middle of our zero-volume 'soma'
+        // corresponds to proximal end of cable.
+        c.add_stimulus({0,0.5}, {0.,  HUGE_VAL, I});
```

Comment:
`stimulus::value_type` is double now, but if that was changed to `float`, `HUGE_VAL` could be a problem.
How about `std::numeric_limits<nest::mc::stimulus::value_type>::max()`?